app/views/dashboards/show.html.erb

```diff
@@ -5,27 +5,28 @@
   <%= link_to "Save a snippet", new_snippet_path %>
   <%= link_to "Save an audio file", new_audio_file_path %>
   <%= link_to "Save a wishlist item", new_wishlist_path %>
-  <h2>🎹 Latest music snippets: 🎸</h2>
+  <h2>🎹 Latest music snippet activity: 🎸</h2>
 </div>
 
+<div class="snippet">
+  <% @activities.each_with_index do |activity, i| %>
```

Comment:
Could this be consolidated with `to_partial_path` by converting this to:

```erb
<div class="snippet">
  <%= render @activities %>
</div>
```

Moving the contents of this block into a partial?